examples/vue/README.md

```diff
@@ -0,0 +1,13 @@
+# Validate field Example
+
+Example integrating `fonk` with `Vue.js 2` at field level.
```

Comment:
Check [validate-field Readme](https://github.com/Lemoncode/fonk/tree/master/examples/vanilla/js/validate-field) to keep same look and feels